CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Improvements
 
+- **Event Hubs Scaler:** Support Azure Active Direcotry Pod & Workload Identity for Storage Blobs ([#3569](https://github.com/kedacore/keda/issues/3569))
```

Comment:
Nit - Scan we alphabetically sort please?

Reply:
> LGTM, @v-shenoy any concerns from your side?

LGTM. Maybe we should have this rebased and run the recently added EventHub e2e tests.